[ERROR] 2017-05-30 14:58:23,909 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 14:58:24,122 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:02:24,444 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:02:24,650 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:06:24,974 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:06:25,092 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:10:25,420 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:10:25,557 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:14:25,886 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:14:26,310 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:18:26,636 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:18:26,750 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:22:27,077 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:22:27,457 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:26:27,784 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:26:27,911 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:30:28,230 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:30:28,737 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:34:29,062 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:34:29,182 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:38:29,511 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:38:29,863 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:42:30,194 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:42:30,337 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:46:30,651 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:46:30,788 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:50:31,116 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:50:31,181 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:54:31,477 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:54:31,684 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 15:58:32,521 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 15:58:32,891 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:02:33,170 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:02:33,369 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:06:33,645 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:06:33,786 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:10:34,067 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:10:34,363 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:14:34,635 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:14:34,766 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:18:35,049 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:18:35,499 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:22:35,779 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:22:36,179 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:26:36,465 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:26:36,536 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:30:36,818 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:30:36,948 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:34:37,221 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:34:37,389 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:38:37,673 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:38:37,808 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:42:38,094 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:42:38,231 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:46:38,519 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:46:38,618 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:50:38,895 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:50:38,964 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:54:39,241 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:54:39,315 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 16:58:39,862 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 16:58:40,292 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:02:40,566 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:02:40,779 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:06:41,061 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:06:41,159 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:10:41,463 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:10:41,654 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:14:41,959 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:14:42,089 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:18:42,397 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:18:42,542 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:22:42,837 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:22:42,963 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:26:43,268 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:26:43,445 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:30:43,750 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:30:43,876 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:34:44,174 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:34:44,315 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:38:44,609 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:38:44,731 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:42:45,035 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:42:45,104 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:46:45,396 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:46:45,648 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:50:45,943 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:50:45,994 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:52:54,299 | /bench-repo/frappe-bench/apps/frappe/frappe/app.py:
Site: site1.local
Form Dict: {
 "action": "Save", 
 "cmd": "frappe.desk.form.save.savedocs", 
 "doc": "{\"naming_series\":\"CSE-.###\",\"parent_webste_route\":\"myprojects\",\"creation\":\"2017-05-24 14:19:06.831494\",\"project_guide\":\"INS/00003\",\"owner\":\"protimr16@gmail.com\",\"modified_by\":\"protimr16@gmail.com\",\"project_desc\":\"\",\"docstatus\":0,\"start_date\":\"2017-05-31\",\"__onload\":{\"is_website_generator\":true,\"published\":true},\"show_in_website\":1,\"doctype\":\"MyProjects\",\"project_title\":\"Network on Chips\",\"name\":\"CSE-001\",\"idx\":0,\"modified\":\"2017-05-24 15:09:40.510327\",\"dept\":\"ECE\",\"__last_sync_on\":\"2017-05-30T12:22:11.094Z\",\"__unsaved\":1,\"instructor_name\":\"Dr. Bidisha Dasgupta\"}"
}
Request Error
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/app.py", line 55, in application
    response = frappe.handler.handle()
  File "/bench-repo/frappe-bench/apps/frappe/frappe/handler.py", line 21, in handle
    data = execute_cmd(cmd)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/handler.py", line 52, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 907, in call
    return fn(*args, **newargs)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/desk/form/save.py", line 22, in savedocs
    doc.save()
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 230, in save
    return self._save(*args, **kwargs)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 263, in _save
    self.run_before_save_methods()
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 771, in run_before_save_methods
    self.run_method("validate")
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 666, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 888, in composer
    hooks.append(frappe.get_attr(handler))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 887, in get_attr
    return getattr(get_module(modulename), methodname)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 672, in get_module
    return importlib.import_module(modulename)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named mynewapp2.random.myprojects
[ERROR] 2017-05-30 17:52:54,482 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2017-05-30 17:52:54.303867-127.0.0.1-3c8
[ERROR] 2017-05-30 17:54:46,293 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:54:46,357 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 17:58:46,688 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 17:58:47,107 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:01:39,275 | /bench-repo/frappe-bench/apps/frappe/frappe/app.py:
Site: site1.local
Form Dict: {
 "action": "Save", 
 "cmd": "frappe.desk.form.save.savedocs", 
 "doc": "{\"naming_series\":\"CSE-.###\",\"parent_webste_route\":\"myprojects\",\"creation\":\"2017-05-24 14:19:06.831494\",\"project_guide\":\"INS/00002\",\"owner\":\"protimr16@gmail.com\",\"modified_by\":\"protimr16@gmail.com\",\"project_desc\":\"\",\"docstatus\":0,\"start_date\":\"2017-05-31\",\"__onload\":{\"is_website_generator\":true,\"published\":true},\"show_in_website\":1,\"doctype\":\"MyProjects\",\"project_title\":\"Network on Chips\",\"name\":\"CSE-001\",\"idx\":0,\"modified\":\"2017-05-24 15:09:40.510327\",\"dept\":\"CSE\",\"__last_sync_on\":\"2017-05-30T12:31:17.167Z\",\"__unsaved\":1,\"instructor_name\":\"Dr. Gautam Barua\"}"
}
Request Error
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/app.py", line 55, in application
    response = frappe.handler.handle()
  File "/bench-repo/frappe-bench/apps/frappe/frappe/handler.py", line 21, in handle
    data = execute_cmd(cmd)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/handler.py", line 52, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 907, in call
    return fn(*args, **newargs)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/desk/form/save.py", line 22, in savedocs
    doc.save()
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 230, in save
    return self._save(*args, **kwargs)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 263, in _save
    self.run_before_save_methods()
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 771, in run_before_save_methods
    self.run_method("validate")
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 666, in run_method
    out = Document.hook(fn)(self, *args, **kwargs)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/model/document.py", line 888, in composer
    hooks.append(frappe.get_attr(handler))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 887, in get_attr
    return getattr(get_module(modulename), methodname)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 672, in get_module
    return importlib.import_module(modulename)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named mynewapp2.random.myprojects
[ERROR] 2017-05-30 18:01:39,505 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2017-05-30 18:01:39.278269-127.0.0.1-22c
[ERROR] 2017-05-30 18:02:47,379 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:02:47,537 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:06:47,816 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:06:47,979 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:10:48,253 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:10:48,414 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:14:48,708 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:14:48,856 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:18:49,151 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:18:49,328 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:22:49,620 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:22:49,882 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:26:50,181 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:26:50,313 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:30:50,604 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:30:50,778 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:34:51,072 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:34:51,218 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-30 18:38:51,521 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-30 18:38:51,764 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:14:20,722 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:14:22,688 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:18:23,008 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:18:23,075 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:22:23,362 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:22:23,495 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:26:23,948 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:26:24,048 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:30:24,361 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:30:24,494 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:34:24,795 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:34:24,917 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:38:25,216 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:38:25,354 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:42:25,645 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:42:25,789 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:46:26,091 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:46:26,266 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:50:26,571 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:50:26,696 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:54:27,246 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:54:28,264 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 12:58:28,729 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 12:58:28,936 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:02:29,231 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:02:29,433 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
